modules/yarpc/yarpc.go

```diff
@@ -259,14 +259,14 @@ func newYARPCModule(
 	host service.Host,
 	reg CreateThriftServiceFunc,
 	options ...ModuleOption,
-) (*YARPCModule, error) {
+) (*Module, error) {
 	moduleOptions := &moduleOptions{}
 	for _, option := range options {
 		if err := option(moduleOptions); err != nil {
 			return nil, err
 		}
 	}
-	module := &YARPCModule{
+	module := &Module{
 		host:        host,
 		statsClient: newStatsClient(host.Metrics()),
 		log:         ulog.Logger(context.Background()).With(zap.String("module", host.Name())),
```

Comment:
I think host.Name() is now scoped post-peter's-refactor to the module name? @peter-edge to confirm